tests/pom.xml

```diff
@@ -34,7 +34,7 @@
                     <useUIThread>false</useUIThread>
                     <providerHint>junit5</providerHint>
                     <systemProperties>
-                        <org.eclipse.swtbot.search.timeout>20000</org.eclipse.swtbot.search.timeout>
+                        <org.eclipse.swtbot.search.timeout>60000</org.eclipse.swtbot.search.timeout>
```

Comment:
I would question this change. If a widget is not present within at most 10/20 seconds for this particular set of tests, there are bigger issues that need to be investigated. A side effect of increasing the timeout is that it holds getting the results of a failing (out of sync test/IDE) run a very long time.